src/components/articles/Article.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { getArticleByID, getCommentsByArticle } from '../../api';
+import { CommentCard } from './CommentCard';
+
+export const Article = () => {
+  const [article, setArticle] = useState({});
+  const [commentList, setCommentList] = useState([]);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    const promiseArr = [
+      getArticleByID(article_id),
+      getCommentsByArticle(article_id),
+    ];
+
+    Promise.all(promiseArr).then((data) => {
+      setArticle(data[0].article);
+      setCommentList(data[1].comments);
+    });
+  }, []);
```

Comment:
article_id should go into dependencies array, so when id changes, the component can be re-rendered